pkg/cmd/version/version.go

```diff
@@ -0,0 +1,61 @@
+package version
```

Comment:
+1 for moving into a dedicated package. It should be an ongoing task to whenever we touch commands to further decompose this massive _cmd_ package.

Reply:
Agreed :100: